ember_debug/object-inspector.js

```diff
@@ -493,6 +509,10 @@ function replaceProperty(properties, name, value, options) {
   let prop = { name, value: inspectValue(value) };
   prop.isMandatorySetter = options.isMandatorySetter;
   prop.readOnly = options.readOnly;
+  prop.dependentKeys = options.dependentKeys || [];
+  let hasServiceFootprint = prop.value && typeof prop.value.inspect === 'string' ? prop.value.inspect.includes('@service:') : false;
+  prop.isService = options.isService || hasServiceFootprint;
+  prop.code = options.code;
```

Comment:
Can you add tests for services and DKs [here](https://github.com/emberjs/ember-inspector/blob/f20a42c7d3836c17c97fc84ac7f413d64659bcd3/tests/ember_debug/object-inspector-test.js)?

Reply:
@teddyzeenny done